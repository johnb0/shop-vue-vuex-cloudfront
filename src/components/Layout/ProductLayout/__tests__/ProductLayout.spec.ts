import { shallowMount } from '@vue/test-utils';
import ProductLayout from '@/components/Layout/ProductLayout/ProductLayout.vue';

describe('HelloWorld.vue', () => {
	it('renders props.msg when passed', () => {
		const wrapper = shallowMount(ProductLayout);

		expect(wrapper.html()).toMatchSnapshot();
	});
});
