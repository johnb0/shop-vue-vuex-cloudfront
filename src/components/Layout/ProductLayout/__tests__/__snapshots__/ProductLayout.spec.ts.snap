// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HelloWorld.vue renders props.msg when passed 1`] = `
<v-app-stub id="app">
  <v-product-header-stub></v-product-header-stub>
  <v-main-stub tag="main">
    <v-container-stub tag="div" fluid="true" class="px-0 py-0"></v-container-stub>
  </v-main-stub>
  <v-footer-stub height="auto" tag="footer" class="white py-12 d-flex flex-column justify-center align-center">
    <p class="text-subtitle1 text--secondary">
      common.thankYouForPurchase
    </p>
    <p class="text-body2 text--secondary">
      Copyright Â©
      app.title
      2022.
    </p>
  </v-footer-stub>
</v-app-stub>
`;
